src/+otp/+pendulum/PendulumProblem.m

```diff
@@ -36,16 +36,18 @@ function onSettingsChanged(obj)
         function validateNewState(obj, newTimeSpan, newY0, newParameters)
             validateNewState@otp.Problem(obj, newTimeSpan, newY0, newParameters)
             
-            %            otp.utils.StructParser(newParameters) ...
-            %                 .checkField('g', 'scalar', 'real', 'finite', 'positive') ...
-            %                 .checkField('lengths', 'vector', 'real', 'finite', 'positive');
+            otp.utils.StructParser(newParameters) ...
+                .checkField('g', 'scalar', 'real', 'finite', 'positive') ...
+                .checkField('lengths', 'vector', 'real', 'finite', 'positive') ...
+                .checkField('masses',  'vector', 'real', 'finite', 'positive');
```

Comment:
would vector still work for a scalar?

Reply:
Yes.  It uses `isvector` which is true even for scalars.